fiboa_cli/datasets/us_usda_cropland.py

```diff
@@ -0,0 +1,96 @@
+"""
+
+EXTENSIONS = ["https://fiboa.github.io/crop-extension/v0.1.0/schema.yaml"]
+PROVIDERS = [
+    {
+        "name": "United States Department of Agriculture",
+        "url": "https://www.nass.usda.gov/",
+        "roles": ["licensor", "producer"]
+    }
+]
+LICENSE ={
+    "title": "The USDA NASS Crop Sequence Boundaries and the data offered at https://www.nass.usda.gov/Research_and_Science/Crop-Sequence-Boundaries are provided to the public as is and are considered public domain and free to redistribute. Users of the Crop Sequence Boundaries (CSB) are solely responsible for interpretations made from these products. The CSB are provided 'as is' and the USDA NASS does not warrant results you may obtain using the data. Contact our staff at (SM.NASS.RDD.GIB@usda.gov) if technical questions arise.",
+    "href": "https://gee-community-catalog.org/projects/csb/#license-and-liability",
+    "type": "text/html",
+    "rel": "license"
+}
+
+COLUMNS = {
+    "geometry": "geometry",
+    "CSBID": "id",
+    'CDL2023': 'crop:code',
+    'crop:name': 'crop:name',
+    'CNTY': "administrative_area_level_2",
+}
+
+ADD_COLUMNS = {
+    "determination_datetime": "2023-05-01T00:00:00Z",
+    "crop:code_list": "https://github.com/fiboa/cli/blob/main/fiboa_cli/datasets/data-files/us_usda_cropland.csv",
+}
+
+MISSING_SCHEMAS = {
+    "properties": {
+        "administrative_area_level_2": {
+            "type": "string"
+        },
+    }
+}
+
+
+def migrate(gdf):
+    # "dissolve": merge adjacent polygons with the same crop
+    # geodataframe.Dissolve(method="unary") is **slow** for large datasets
+    # So we're handling this huge dataset in blocks, states are a natural grouping-method
+    states = list(gdf["STATEFIPS"].unique())
+    gdfs = []
+    for state in states:
+        log(f"Handling State {state}", "info")
+        df = gdf[gdf["STATEFIPS"] == state].explode()
+        df.dissolve(by=["CDL2023"], aggfunc="first").explode()
+        gdfs.append(df)
+    gdf = pd.concat(gdfs)
+    del gdfs
+    mapping = load_ec_mapping(url=join(dirname(__file__), "data-files", "us_usda_cropland.csv"))
+    original_name_mapping = {int(e["original_code"]): e["original_name"] for e in mapping}
+    gdf['crop:name'] = gdf['CDL2023'].map(original_name_mapping)
+    return gdf
+
+
+def convert(output_file, cache = None, **kwargs):
+    convert_(
+        output_file,
+        cache,
+        SOURCES,
+        COLUMNS,
+        ID,
+        TITLE,
+        DESCRIPTION,
+        migration=migrate,
+        column_additions=ADD_COLUMNS,
+        extensions=EXTENSIONS,
+        providers=PROVIDERS,
+        license=LICENSE,
+        explode_geometries=True,
```

Comment:
```suggestion
        explode_geometries=True,
```

Applied automatically.